bigcache.go

```diff
@@ -32,9 +32,9 @@ type Response struct {
 type RemoveReason uint32
 
 const (
+	_ RemoveReason = iota
 	// Expired means the key is past its LifeWindow.
-	// @TODO: Go defaults to 0 so in case we want to return EntryStatus back to the caller Expired cannot be differentiated
-	Expired RemoveReason = iota
+	Expired
```

Comment:
looks like a breaking change

Reply:
yes, it's a breaking change. The issue is discussed in #181, and hopefully in v3.0

Reply:
I think I would like to see this explicitly defined as a numerical instead of an iota. That forces the API to remain stable, in my opinion.

Reply:
@mxplusb updated, please take a look.